### DISCUSSION TITLE
Affichage de l'alert si highPrivacy et mandatory = true

### DISCUSSION THREAD

**Discussion by gregdeluxee** [AmauriC/tarteaucitron.js#843] [Ideas]
Bonjour,

Actuellement le paramètre mandatory permet d'afficher un message concernant les cookies de fonctionnement dans la popup.
Cependant, si aucun service n'est utilisé sur le site, l'alerte(bannière) ne sera jamais affichée pour que l'utilisateur en soit conscient.
Or cela reste un cookie et il est obligatoire d'en informer l'utilisateur.

Dans le cas où highPrivacy = true et mandatory = true, ce serait logique d'afficher l'alerte.

Qu'en pensez-vous ?

Pour info, j'ai réussi à forcer son affichage en modifiant le code mais le problème est que le bouton "Tout accepter" ne réagit pas. "Personnaliser" ouvre bien la popup.
La difficulté, c'est que l'affichage de l'alerte se fait dépendamment des services/jobs

**Comment by AmauriC** [comment]
Oui et non. Si il n'y a que des cookies essentiels (panier, espace membre, analytic anonyme, ...) ce n'est pas utile d'ennuyer l'utilisateur avec un bandeau juste informatif et ce n'est pas du tout obligatoire.

Si vous voulez quand même afficher un message, je ne pense pas que ce soit le role de tarteaucitron.
